packages/frontend-2/components/viewer/selection/Sidebar.vue

```diff
@@ -205,36 +204,30 @@ onKeyStroke('Escape', () => {
 })
 
 watch(
-  () => objects.value.length,
-  (newLength) => {
-    // Dont open sidebar if a comment is open
-    if (newLength !== 0 && !focusedThreadId.value) {
-      sidebarOpen.value = true
-    } else if (newLength === 0) {
+  [
```

Comment:
watchers were conflicting with each other and race conditions were happening, so i combined them all into 1 watcher